packages/shared/src/contexts/ActiveFeedNameContext.tsx

```diff
@@ -0,0 +1,11 @@
+import { createContext } from 'react';
+import { AllFeedPages } from '../lib/query';
+
+export type ActiveFeedNameContextValue = {
+  feedName?: AllFeedPages;
+  setFeedName: (feedName: AllFeedPages) => void;
```

Comment:
This feels like anti-pattern of context. Context feed value should be set through context Provider on lower levels if needs to change. So I don't think `setFeedName` needs to be exposed to child components.

Reply:
At the moment we don't need it actually, will remove. I added when I was thinking of refactoring, and feedName seemed to be changed in different places for different reasons, so setup this as a way to do it (not sure why needed for this particular case).

In general I don't see it as an anti-pattern though, I've seen this quite a lot used for theme context for ex, where a button somewhere in the child components need to have access to changing the context.